compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.td

```diff
@@ -105,7 +105,7 @@ def LLVMCPUSplitReduction : Pass<"iree-llvmcpu-split-reduction", "func::FuncOp">
   let constructor = "mlir::iree_compiler::createLLVMCPUSplitReductionPass()";
   let options = [
     Option<"enableFpReductionReordering", "enable-fp-reduction-reordering",
-      "bool", /*default=*/"false",
+      "bool", /*default=*/"true",
```

Comment:
Not sure how this change is related to this patch.

Reply:
Reverted this.